### PR TITLE
Getting app to load

### PR DESCRIPTION
**Pull request by intricatecloud**
Hi!

I cloned your project and made a couple of edits to get it to work - I've been able to sign in so I can see you've done at least that part of the assignment. This Pull Request contains changes that you can "pull" into your project - click the Merge Pull Request button to apply these changes to your project. You can then `git pull` to pull in the latest changes from your repository.

You can now move on to the hosting step of Assignment #3 to deploy this to your own domain.

Whatever you did with Firebase Hosting changed your index.html file and your package.json and it interfered with the way the React app was loading so it only showed the "Welcome to Firebase Hosting" screen.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -49,5 +49,5 @@
   "bugs": {
     "url": "https://github.com/CassiniXu/ryan/issues"
   },
-  "homepage": "https://github.com/CassiniXu/ryan#readme"
+  "homepage": "/"
```
Comment (intricatecloud) [inline package.json:52]: This line kept causing the URL to app to load as `localhost:3000/CassiniXu/rya` when it launched the browser, and that caused other loading problems.

File: public/index.html [hunk 2]
```diff
@@ -1,89 +1,43 @@
 <!DOCTYPE html>
-<html>
```
Comment (intricatecloud) [inline public/index.html:2]: I reset this file back to the default from create-react-app.

File: package.json (hunk 1 above)
Comment (CassiniXu) [reply package.json:52]: Thank you very much! Is github who changed this line to be the link to readme?

Comment (CassiniXu) [reply package.json:52]: > This line kept causing the URL to app to load as `localhost:3000/CassiniXu/rya` when it launched the browser, and that caused other loading problems.

Thank you very much! Is github who changed this line to be the link to readme?

File: public/index.html (hunk 2 above)
Comment (CassiniXu) [reply public/index.html:2]: @intricatecloud Thanks! Do I need to reset all the index.html to the default everytime I start a react app

**Pull request merged**